packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.tsx

```diff
@@ -49,11 +50,17 @@ export class MessageBar extends React.Component<IMessageBarProps, IMessageBarSta
     /**
      * Live regions need an update to announce content.
      */
-    setTimeout(() => {
+    this._mountTimeout = setTimeout(() => {
```

Comment:
@mdahamiwal The timeout should be moved to componentDidMount. You should only be calling setState while mounted.